yt/utilities/exceptions.py

```diff
@@ -176,8 +176,8 @@ def __init__(self, ds):
 
     def __str__(self):
         return (
-            "Simulation %s has no stopping condition.  StopTime or StopCycle should be set."
-            % self.ds
+            "Simulation %s has no stopping condition. "
+            "StopTime or StopCycle should be set." % self.ds
```

Comment:
Maybe this should be changed to match @cphyc's comment about string formatting in another place?

Reply:
definitely

Reply:
Actually this line is going to be conflicted no matter how we handle it. I think the best route is to merge Corentin's flynt PR first and I'll resolve the conflict here with an string.

Reply:
(just noticed that the branch has conflicts already, so might as well get them all in one go)

Reply:
Totally makes sense! We'll hold off on merging this until the flynt PR(s) go in.